utils/heatmap_dataset.py

```diff
@@ -1,14 +1,15 @@
 import torch
-from torch.utils.data import Dataset
-from torch_em.util import ensure_spatial_array, ensure_tensor_with_channels
+# from torch.utils.data import Dataset
+from torch_em.util import ensure_tensor_with_channels
 import numpy as np
 
 import zarr
 import os
 
 from data_processing.create_heatmap import process_tomogram
 
-class HeatmapLoader(torch.utils.data.Dataset):
+
+class HeatmapDataset(torch.utils.data.Dataset):
```

Comment:
This is a Dataset, not a Dataloader, so it should be named accordingly. See e.g. https://pytorch.org/tutorials/beginner/basics/data_tutorial.html for the distinction.

Reply:
Also, we should refactor this loader a bit, but we can discuss how to do this later. I think the best option would be to inherit from https://github.com/constantinpape/torch-em/blob/main/torch_em/data/segmentation_dataset.py .